test/shell/pvmove-raid-segtypes.sh

```diff
@@ -46,6 +46,15 @@ pvmove $mode -n $lv1 "$dev5" "$dev4"
 check lv_tree_on $vg $lv1 "$dev2" "$dev4"
 check lv_tree_on $vg ${lv1}_foo "$dev5"
 check dev_md5sum $vg $lv1
+
+# Check moving SubLVs one-by-one works
+pvmove $mode -n ${lv1}_rimage_0 "$dev4" "$dev1"
+check lv_tree_on $vg $lv1 "$dev1" "$dev2" "$dev4"
+check dev_md5sum $vg $lv1
+pvmove $mode -n ${lv1}_rmeta_0 "$dev4" "$dev1"
+check lv_tree_on $vg $lv1 "$dev1" "$dev2"
+check dev_md5sum $vg $lv1
+
```

Comment:
I haven't actually run these tests yet, since I need to set up a special test environment.
Will probably get around to doing it in a few days' time, unless a maintainer can graciously run the tests for me :)

Reply:
Ok, actually tested to work now